doc/cephfs/quota.rst

```diff
@@ -74,6 +74,12 @@ To set a quota::
   setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
   setfattr -n ceph.quota.max_files -v 10000 /some/dir         # 10,000 files
 
+``ceph.quota.max_bytes`` can also be set using human readable format
+(k|K / m|M / g|G / t|T)::
+
+  setfattr -n ceph.quota.max_bytes -v 100k /some/dir          # 100 MB
```

Comment:
s/100k/100M/
s/100MB/100MiB/

Reply:
ah this is what i was talking about, cool will update this

Reply:
done.